pages/confirm-installation-popup.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class ConfirmInstallationPopup {
+    readonly page: Page;
+    readonly confirmButton: Locator;
+    readonly cancelButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.confirmButton = page.getByRole('button', { name: 'Continue' });
+        this.cancelButton = page.getByRole('button', { name: 'Cancel' });
+    }
+
+    async confirm() {
+        await this.confirmButton.click();
+    }
+
+    async cancel() {
+        await this.cancelButton.click();
+    }
+
+}
```

Comment:
by default vscode leave a blank line at the end of the file, let's follow that as it is the tool that better integrates with Playwright.